redirects-transition.conf

```diff
@@ -309,14 +309,37 @@ rewrite ^/info/hearings.shtml https://www.fec.gov/meetings/?tab=hearings redirec
+rewrite ^/info/mis-sc.htm https://www.fec.gov/resources/record/1996/jan96.pdf#page=9 redirect;
+rewrite ^/info/nprmintr.htm https://sers.fec.gov/fosers/showpdf.htm?docid=62698 redirect;
 rewrite ^/info/outreach.shtml https://www.fec.gov/help-candidates-and-committees/trainings/ redirect;
+rewrite ^/info/pacs.shtml https://www.fec.gov/help-candidates-and-committees/ redirect;
+rewrite ^/info/parties.shtml https://www.fec.gov/help-candidates-and-committees/ redirect;
+rewrite ^/info/plainwriting.shtml https://www.fec.gov/about/plain-language/ redirect;
+rewrite ^/info/plainwriting041312.shtml https://www.fec.gov/about/plain-language/ redirect;
+rewrite ^/info/plainwriting041313.shtml https://www.fec.gov/about/plain-language/ redirect;
+rewrite ^/info/plainwriting071311.shtml https://www.fec.gov/about/plain-language/ redirect;
+rewrite ^/info/plainwritingreport.shtml https://www.fec.gov/about/plain-language/ redirect;
+rewrite ^/info/prim98.htm https://www.fec.gov/resources/record/1998/jan98.pdf redirect;
+rewrite ^/info/primref.htm https://www.fec.gov/resources/record/1996/jan96.pdf redirect;
 rewrite ^/info/publications.shtml https://www.fec.gov/help-candidates-and-committees/guides redirect;
+rewrite ^/info/recent_conference_materials.shtml https://www.fec.gov/help-candidates-and-committees/trainings/ redirect;
 rewrite ^/info/report_dates.shtml https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/ redirect;
+rewrite ^/info/report_dates_midterm6.shtml https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/ redirect;
+rewrite ^/info/report_dates_midterm.shtml https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/ redirect;
+rewrite ^/info/report_dates_non.shtml https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/ redirect;
+rewrite ^/info/report_dates_non7.shtml https://www.fec.gov/help-candidates-and-committees/dates-and-deadlines/ redirect;
+rewrite ^info/sbrefa.htm https://www.fec.gov/help-candidates-and-committees/guides/ redirect;
+rewrite ^info/sd-w.htm https://www.fec.gov/resources/record/1996/jan96.pdf redirect;
+rewrite ^/info/subj98.htm https://www.fec.gov/legal-resources/court-cases/ redirect;
 rewrite ^/info/TimelyTipsArchive.shtml https://www.fec.gov/updates/?update_type=tips-for-treasurers redirect;
 rewrite ^/info/TimelyTipsArchive2006.shtml https://www.fec.gov/updates/?update_type=tips-for-treasurers redirect;
 rewrite ^/info/TipsforTreasurers.shtml https://www.fec.gov/updates/?update_type=tips-for-treasurers redirect;
+rewrite ^/info/tips4treasurers.xml https://www.fec.gov/updates/?update_type=tips-for-treasurers redirect;
 rewrite ^/info/toolkit.shtml https://www.fec.gov/help-candidates-and-committees/ redirect;
+rewrite ^info/treas.htm https://www.fec.gov/updates/committee-treasurers-2017-record/ redirect;
+rewrite ^info/tresur https://www.fec.gov/updates/committee-treasurers-2017-record/ redirect;
 
```

Comment:
Hi Brad - for the two info ones, need a / before info

Reply:
Just caught this; it was a mistake in our spreadsheet. For line 342, it should be ^info/tresur.htm

Reply:
Corrected these two issues in my second commit.